### PR TITLE
Add discord icon

### PR DESCRIPTION
**Pull request by Varunram**
Didn't work as intended, I got an arc icon, but must be easy for @Barizpwnz to edit it from here :)

### CODE REVIEW

**Comment by Varunram** [comment]
@jjenzz @Barizpwnz Please review :)

**Comment by jjenzz** [comment]
Deploy preview ready!

Built with commit dd7dd5742093640e32c712fb87ee27b254de33d4

https://deploy-preview-69--musicoin-frontend.netlify.com

**Comment by Barizpwnz** [comment]
https://www.dropbox.com/sh/t35jhrf1bcfkraw/AACvSY4vv3sMGjKyzm_PnAaVa?dl=0

I change all active icons color, and discord icons : discord_white.svg   discord_yellow.svg

**Comment by Varunram** [comment]
Awesome, once Jenna approves stuff, we can merge them.

File: src/components/icons/Discord.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import Icon from '.';
+
+export default class Discord extends Component {
+	render() {
+		return (
+			<Icon {...this.props} viewBox="0 0 24 24">
+				<title>Discord</title>
+				<g stroke="none" strokeWidth="1" fillRule="evenodd" fill="none">
+					<path d="M106.8,84.1 C101.1,84.1 96.6,89 96.6,95.1 C96.6,101.2 101.2,106.1 106.8,106.1 C112.5,106.1 117,101.2 117,95.1 C117,89 112.4,84.1 106.8,84.1 Z M70.3,84.1 C64.6,84.1 60.1,89 60.1,95.1 C60.1,101.2 64.7,106.1 70.3,106.1 C76,106.1 80.5,101.2 80.5,95.1 C80.6,89 76,84.1 70.3,84.1 Z" />
```
Comment (Varunram) [inline src/components/icons/Discord.js]: This svg was obtained from font-awesome's pre release but does not render properly. @jjenzz any ideas why?

Comment (jjenzz) [reply src/components/icons/Discord.js]: Is there anyway I can see the original SVG that you copied this from?

Comment (Varunram) [reply src/components/icons/Discord.js]: Oh, its the same one we use on the main site right now, actually. Its present at discord.svg over at images in that repo.

**Pull request merged**